src/twenty-three-video/__tests__/twenty-three-video.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import { TwentyThreeVideoProps } from "../types";
+import TwentyThreeVideo from "../twenty-three-video";
+
+const mockData = require("./mock.json");
```

Comment:
Please comment what the mockdata roughly contains. In this case it's good to know that it only id's so that when I read the example1 object I know that the remaining properties should cover all the configuration necessary for the component to work.